egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1c.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# same as 1a, but with more TDNN layers between each LSTM
+#System            1a        1c
```

Comment:
adding more TDNNs in between lstms improves results while adding TDNNs after LSTMs made it worse.